app/config/config.yml

```diff
@@ -26,6 +26,44 @@ framework:
     fragments:       ~
     http_method_override: true
 
+nelmio_security:
+    clickjacking:
+        paths:
+            '^/.*': DENY
+    external_redirects:
+        abort: true
+        log: true
+        whitelist:
+            - %saml_remote_idp_host%
+
+    csp:
+        report:
+            report-uri: [/nelmio/csp/report]
```

Comment:
Do you have a controller set up for these CSP reports (see links below)? Can there be multiple report URLs? Also, do we only enforce `script-src: [self]` (JavaScript)? What about `default-src: [self]` (everything)?

https://github.com/SURFnet/Stepup-SelfService/blob/d415fc9e81d1b0663a85f867b10ce8461345aee3/app/config/config.yml#L71
https://github.com/SURFnet/Stepup-SelfService/blob/1ba8b776aff73aaa490058420de06d5e105c0206/app/config/routing.yml#L6